### PR TITLE
tartube: init at 2.0.016

### LINKED ISSUE #84655: Tartube
**Project description**

Tartube is a GUI front-end for youtube-dl, partly based on youtube-dl-gui and written in Python 3 / Gtk 3.

**Metadata**

* homepage URL: https://tartube.sourceforge.io/
* source URL: https://github.com/axcore/tartube
* license: gpl3
* platforms: linux

### PR DESCRIPTION
**Pull request by mkg20001**
###### Motivation for this change

Fix https://github.com/NixOS/nixpkgs/issues/84655
This work uses @pbogdan's expression from https://github.com/NixOS/nixpkgs/issues/84655#issuecomment-614146162

@pbogdan: I didn't find any issues with the app, it seems to work. I also have made you the author of the commit, since I believe you should get the honor of being the commit author. Just comment if you don't like that.

I've added myself and @luc65r as a maintainer - I've asked him about that and he aggreed

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by pbogdan** [comment]
Sounds good to me, thank you! And just a heads up - looks like `gdk_pixbuf` needs to be swapped for `gdk-pixbuf` to make the CI check happy as it doesn't accept aliases.

File: pkgs/applications/video/tartube/default.nix [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  makeWrapperArgs = [
+    "--prefix PATH : ${stdenv.lib.makeBinPath [ youtube-dl ]}"
+  ];
+
+  meta = {
+    description = "A GUI front-end for youtube-dl";
+    license = lib.licenses.gpl3;
+    platforms = lib.platforms.linux;
+    maintainers = with lib.maintainers; [ mkg20001 luc65r ];
+  };
```
Comment (worldofpeace) [inline pkgs/applications/video/tartube/default.nix:66]:
```suggestion
  meta = with lib; {
    description = "A GUI front-end for youtube-dl";
    license = licenses.gpl3;
    platforms = platforms.linux;
    maintainers = with maintainers; [ mkg20001 luc65r ];
  };
```

File: pkgs/applications/video/tartube/default.nix [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  strictDeps = false;
+
+  propagatedBuildInputs = with python3Packages; [
+    moviepy
+    pygobject3
+    pyxdg
+    requests
+  ];
+
+  buildInputs = [
```
Comment (worldofpeace) [inline pkgs/applications/video/tartube/default.nix:41]: adding `glib`

File: pkgs/applications/video/tartube/default.nix [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  version = "2.0.016";
+
+  src = fetchFromGitHub {
+    owner = "axcore";
+    repo = "tartube";
+    rev = "v${version}";
+    sha256 = "1y77ykihyi4v6xlsm5xldbs9lzq229l574rxz6qfvrjcbbwajfj9";
+  };
+
+  patches = [];
```
Comment (worldofpeace) [inline pkgs/applications/video/tartube/default.nix:25]:
```suggestion
```

File: pkgs/applications/video/tartube/default.nix (hunk 1 above)
Comment (worldofpeace) [reply pkgs/applications/video/tartube/default.nix:66]: also missing `homepage`

**Review by worldofpeace — APPROVED** [review]
LGTM, works.

**Comment by worldofpeace** [comment]
Also verified we have all typelibs by looking at `gi.repository` imports.

**Comment by worldofpeace** [comment]
Thanks :sparkles:

**Pull request merged**